hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java

```diff
@@ -264,4 +264,36 @@ public static synchronized ServerOptions instance() {
                     disallowEmpty(),
                     true
             );
+
+    public static final ConfigOption<String> ARTHAS_TELNET_PORT =
+        new ConfigOption<>(
+            "arthas.telnet_port",
+            "arthas provides telnet ports to the outside",
+            disallowEmpty(),
+            "8562"
+        );
+
+    public static final ConfigOption<String> ARTHAS_HTTP_PORT =
+        new ConfigOption<>(
+            "arthas.http_port",
+            "arthas provides http ports to the outside",
+            disallowEmpty(),
+            "8561"
+        );
+
+    public static final ConfigOption<String> ARTHAS_IP =
+        new ConfigOption<>(
+            "arthas.ip",
+            "arthas bound ip",
```

Comment:
can we improve description style

Reply:
more desc detail？ i dont know what means 'style' , sorry.